src/pages/homeLayout/service/service.tsx

```diff
@@ -1,11 +1,75 @@
-import Link from "next/link";
+import Image from "next/image";
+
+import mainPagePicture from "../../../assets/pages/mainPage/mainPagePicture.png";
+import mainPagePictureTablet from "../../../assets/pages/mainPage/mainPagePictureTablet.png";
+import mainPagePictureMobile from "../../../assets/pages/mainPage/mainPagePictureMobile.png";
+import listPoint from "../../../assets/pages/mainPage/listPoint.svg";
+
+import styles from "./service.module.scss";
 
 function Service() {
 	return (
-		<div>
-			Service
-			<Link href={"/profitMoney"}>profitMoney</Link>
-		</div>
+		<>
+			<div className={styles.mainPageContainer}>
+				<div className={styles.infoContainer}>
+					<h1 className={styles.infoContainer__title}>
+						Наслаждайся расходами, <br />
+						экономь без усилий
+					</h1>
+					<p className={styles.infoContainer__subtitle}>
+						FREEnance — онлайн-приложение <br />
+						для учёта личных и семейных финансов.
+					</p>
+					<div className={styles.listOfFeatures}>
+						<p className={styles.listOfFeatures__subtitle}>С нами вы сможете легко:</p>
+						<ul className={styles.listOfFeatures__ul}>
```

Comment:
styles.featuresListWrap
styles.featuresListWrap__title
styles.featuresListContainer
styles.featuresListContainer__featureElement
using tag names in jsx class naming is antipattern.  you can always use wrap/container/content name example model for solve this issue